Assets/Capstone/Scripts/Environmental/Sun.cs

```diff
@@ -4,39 +4,60 @@
 
 public class Sun : MonoBehaviour
 {
+    //player refs
     GameObject _player;
     ThirdPersonController tpc;
+    //day counters
     public int yesterday, dayCounter = 0;
 
-    public GameObject waterDay, waterNight;
-    public float rotationSpeed = 10;
+    //rotation speed refs
     public Transform rotation;
+    public float rotationSpeed = 10, sleepRotation, normalRotation;
+   
+    //angle of sun's rotation
+    public float angleInDegrees;
+    //time state bools
+    public bool isMorning, isMidday, isDusk, isNight;

```

Comment:
Maybe consider using an enum here instead of having 4 variables that represent the time.